src/android/test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * test module
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { View } from './view';
+
+export default class Test {
+
+	constructor (app, module) {
+		this.app = app;
+		this.module = module;
+	}
+
+	startup () {
+		this.app.logger.info(this.module.id + ' startup');
+	}
+
+	shutdown () {
+		this.app.logger.info(this.module.id + ' shutdown');
+	}
+
+	static createView (opts = {}) {
```

Comment:
I don't understand why we have this create function. Why not just re-export `View` from `test.js`?

```javascript
export { View } from './view';
```

So instead of doing:

```javascript
import Test from './test';
const myView = Test.createView({});
```

You'd just be doing:

```javascript
import { View } from './test';
const myView = new View({});
```

It's probably better to be more explicit and not introduce an unnecessary function call to the stack.